### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Danik1601** [iongion/container-desktop#91] [Ideas]
Do you plan to translate the app in different languages? Personally I have no interest in this since I predominantly use English language in any kind of software but I know that many people will love that.

If you have such plans, I could help with Russian translation at some point. Russian is highly spoken in Russia, Belarus, Ukraine, Kazakhstan, Kyrgyzstan and other countries of the former USSR (except Baltic countries).

I would love to donate, but PayPal and international payments are almost completely blocked in my country after recent events so I decided that I can help the project the other way.

**Comment by iongion** [comment]
Sure, the project is built since inception with translation support in mind (see https://github.com/iongion/podman-desktop-companion/blob/main/src/app/package.json#L70)
I absolutely planed to add multi-language support, I have lots of former colleagues, friends and contacts from those parts of the planet too.

Priority was functionality, adding translations is clearly not a single person task and contributions will be appreciated, I only know English, French and Romanian, but idiomatic translation needs domain understanding and context awareness, can't just throw a google/bing translate over them, that is easy, but not cool. So better no translations for now.

The international mess we live in due to recent events affects us all, worldwide, don't worry about donations, worry about friends, family and justice, if this project improves your daily activities to help you gain more time for them, that is the biggest win!
